HelpSource/Reference/ServerPluginAPI.schelp

```diff
@@ -0,0 +1,189 @@
+title:: Server Plugin API
```

Comment:
Don't feel discouraged by this type of comments. Your additions will help many until somebody steps up who actually *works* for the even better alternative. We need more of your constructive, diligent and competent attitude, rather than denigrating comments in lazy language. Your additions are very welcome!